source/Government.cpp

```diff
@@ -164,6 +164,18 @@ void Government::Load(const DataNode &node)
 		{
 			if(key == "provoked on scan")
 				provokedOnScan = false;
+			else if(key == "reputation")
+			{
+				for(const DataNode &grand : child)
+				{
+					const string &grandKey = grand.Token(0);
+					bool hasGrandValue = grand.Size() >= 2;
+					if(grandKey == "max" && hasGrandValue)
+						reputationMax = numeric_limits<double>::max();
+					else if(grandKey == "min" && hasGrandValue)
+						reputationMin = numeric_limits<double>::lowest();
```

Comment:
Realized that this is a removal, and so we don't need values.
```suggestion
					if(grandKey == "max")
						reputationMax = numeric_limits<double>::max();
					else if(grandKey == "min")
						reputationMin = numeric_limits<double>::lowest();
```